index.bs

```diff
@@ -1736,240 +1736,210 @@ whether they are exceptions, promise rejection values, or properties.
+    in device ids exposed to the web plaform.
+    Identifiable information includes
+    branding, make and model numbers, etc
+    You can usually use a random number
+    or a unique id instead.
+    Make sure that your ids aren't guessable,
+    and aren't re-used.
 : Keep the user in control
-:: Any existing device ids
-    mapped to or stored with the current session by the user agent
-    should be cleared
-    when users elect to "clear their cookies" (and other related settings).
-    Above all,
-    the user should be in control of this potential tracking state
-    and be able to reset it on demand.
+:: When the user chooses to clear browsing data,
+    make sure any stored device ids are cleared.
 : Hide sensitive ids behind a user permission
-:: Where device identification does not make sense to be expressed in an anonymous way,
-    access to the identifier
-    should be limited by default.
-    One way to limit exposure
-    is to only surface the identifier to author code
-    <em>after</em> obtaining permission from the user.
+:: If you can't create a device id in an anonymous way,
+    limit access to it.
+    Make sure the user can provide
+    [[#consent|meaningful consent]]
+    to a Web page accessing this information.
 : Tie ids to the same-origin model
-:: Identifiers should be unique
-    to the origin of the web content that is attempting to access them.
-    Web content from one origin should receive an identifier that is distinct
-    from the identifier given to web content from any other origin
-    despite the physical device being the same.
-:: Within an origin,
-    ids may have a stable representation to web content.
-    This ensures a consistent developer experience
-    if requesting the same device twice.
+:: Create distinct unique ids for the same physical device
+    for each origin that has has access to it.
+:: If the same device is requested more than once
+    by the same origin, return the same id for it
+    (unless the user has cleared their browsing data).
+    This allows authors to avoid having
+    several copies of the same device.
 : Persistable when necessary
-:: Some device identifiers
-    can only be obtained
-    via a complex or time-consuming device selection process.
-    In such cases,
-    it may make sense for author code
-    to be able to persist such identifiers
-    for use in a later session
-    in order to avoid the selection process a second time.
-    When this is the case,
-    the API should not only
-    provide a stable id during the session for the given origin,
-    but it should also be capable of
-    deterministically producing the same id
-    in subsequent sessions.
+:: If a device id is time consuming to obtain,
+    make sure authors can store an id generated in one session
+    for use in a later session.
+    You can do this by making sure that
+    the procedure to generate the id
+    deterministcially produces the same id
```

Comment:
"deterministically"

Reply:
Also, while a common CS term, it's also a pretty hard word.

Reply:
Changed to "consistently"